cfme/tests/automate/test_file_import.py

```diff
@@ -0,0 +1,34 @@
+from collections import namedtuple
+
+import pytest
+
+from cfme import test_requirements
+
+pytestmark = [test_requirements.automate, pytest.mark.tier(3)]
+
+DatastoreImport = namedtuple("DatastoreImport", ["file_name", "from_domain", "to_domain"])
```

Comment:
I meant it should be defined in the global fixture - where its easier for other test modules to locate and import it. :)

Reply:
ok